src/mutable_data.rs

```diff
@@ -996,28 +990,26 @@ impl SeqEntryActions {
     /// Requires the new `version` of the sequenced entry content. If it does not match the current
     /// version + 1, an error will be returned.
     pub fn del(mut self, key: Vec<u8>, version: u64) -> Self {
-        let _ = self.actions.insert(key, SeqEntryAction::Del(version));
+        let _ = self.0.insert(key, SeqEntryAction::Del(version));
         self
     }
 }
 
 impl From<SeqEntryActions> for BTreeMap<Vec<u8>, SeqEntryAction> {
     fn from(actions: SeqEntryActions) -> Self {
-        actions.actions
+        actions.0
     }
 }
 
 impl From<BTreeMap<Vec<u8>, SeqEntryAction>> for SeqEntryActions {
```

Comment:
It looks like a `From` impl similar to this one is missing for `UnseqEntryActions`, can you please add it?